modules/cnv_introduction_to_cnv.adoc

```diff
@@ -0,0 +1,23 @@
+[[introduction-to-container-native-virtualization]]
+== Introduction to Container-native Virtualization
+
+Container-native Virtualization is an add-on to OpenShift Container
```

Comment:
@aburdenthehand - use the productname variable as defined in the cnv_document_attributes.adoc file.